<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesetGroupEditor</class>
 <widget class="QDialog" name="TilesetGroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>450</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Tileset Group editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>":/engines/moondust/48.png"</normaloff>":/engines/moondust/48.png"</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,100,0" columnstretch="0,0,0,0,0,0,0">
   <item row="3" column="2">
    <widget class="QPushButton" name="tilesetUp">
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="5" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QSpinBox" name="orderWeight">
       <property name="toolTip">
        <string>Defines the custom order priority. If weight values are equal between of different tileset group or equal to -1, tileset groups will be ordered alphabetically.</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="tilesetGroupName"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="category">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="orderWeightLabel">
       <property name="text">
        <string>Order weight</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="groupTitleLabel">
       <property name="text">
        <string>Tileset group name:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="categoryWeight">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="categoryWeightLabel">
       <property name="text">
        <string>Category order weight</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="Open">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="Close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Tilesets list:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="5" colspan="2">
    <widget class="QListWidget" name="tilesetList">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string notr="true">tileset1.ini</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">tileset2.ini</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">tileset3.ini</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="0" rowspan="2" colspan="7">
    <widget class="QGroupBox" name="PreviewBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="tilesetDown">
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QPushButton" name="RemoveTileset">
     <property name="text">
      <string>Remove tileset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QPushButton" name="addTileset">
     <property name="text">
      <string>Add tileset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" rowspan="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="Save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
